src/SWP/Bundle/ContentBundle/Tests/Controller/RouteControllerTest.php

```diff
@@ -51,12 +48,13 @@ public function testCreateEmptyContentRoutesApi()
         ]);
 
         $this->assertEquals(201, $client->getResponse()->getStatusCode());
-        $this->assertArraySubset(json_decode('{"id":"\/swp\/default\/routes\/simple-test-route","content":null,"static_prefix":null,"variable_pattern":null,"name":"simple-test-route","children":[],"id_prefix":"\/swp\/default\/routes","template_name":null,"type":"content","_links":{"self":{"href":"\/api\/v1\/content\/routes\/\/simple-test-route"}}}', true), json_decode($client->getResponse()->getContent(), true));
+        $this->assertArraySubset(json_decode('{"id":"\/swp\/123456\/123abc\/routes\/simple-test-route","content":null,"static_prefix":null,"variable_pattern":null,"name":"simple-test-route","children":[],"id_prefix":"\/swp\/123456\/123abc\/routes","template_name":null,"type":"content","_links":{"self":{"href":"\/api\/v1\/content\/routes\/\/simple-test-route"}}}', true), json_decode($client->getResponse()->getContent(), true));
```

Comment:
Wouldn't it be more readable to test the decoded content? For example, $this->assertEquals($content['content'], null), etc. Maintaining these long strings is a pain.